pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -157,7 +158,7 @@ def __init__(
         # 1. Parsing flags
         # Get registered strategies, built-in accelerators and precision plugins
         self._registered_strategies = StrategyRegistry.available_strategies()
-        self._accelerator_types = ("tpu", "ipu", "gpu", "cpu")
+        self._accelerator_types = AcceleratorRegistry.available_accelerators()
```

Comment:
is this used when "auto" is selected ? if so, we need to be very careful about the ordering inside of the registry